README.md

```diff
@@ -16,8 +16,8 @@ The project is now running at [localhost:8080](http://localhost:8080)
 
 Prerequisites:
 
-* PostgreSQL 11 or higher with PostGIS extension
-* Python 3.8 or higher
+* PostgreSQL 13 or higher with PostGIS extension
+* Python 3.9 or higher
```

Comment:
Are these correct? Compose file specifies PostgreSQL ver. 16 and the CI workflow uses Python 3.12.

Reply:
Updated to use PostgreSQL version 16 and Python 3.12.